lib/zendesk_api/middleware/request/upload.rb

```diff
@@ -18,6 +20,18 @@ def call(env)
 
         private
 
+        def body_for(env)
+          if env[:body].is_a?(Faraday::Multipart::CompositeReadIO)
```

Comment:
For the record, using `is_a?` is a considered a bad practice, see "duck typing", but as we don't control the response, I did not see a quick way around.

https://medium.com/@callie_buruchara/lessons-in-ruby-oop-duck-typing-2ced1809ca76